analysis/test_data_utils.py

```diff
@@ -56,14 +56,37 @@ def test_validate_data_empty():
         data_utils.validate_data(experiment_df)
 
 
+def test_drop_fuzzer_benchmark_trials_above_max():
+    """Tests that drop_fuzzer_benchmark_trials_above_max drops trials for a
+    fuzzer-benchmark if they are beyond the maximum number of trials per
+    fuzzer-benchmark that we specify."""
+    experiment_df = create_experiment_data()
+    new_experiment_df = data_utils.drop_fuzzer_benchmark_trials_above_max(
+        experiment_df, 1)
+    # Sanity check test.
+    assert len(new_experiment_df) < len(experiment_df)
```

Comment:
I'm not sure i understand this test.
Isn't what we want to check is something like:
`new_experiement_df.groupby('benchmark', 'fuzzer')['trial_id'].nunique() == 1` for all rows?